pipelines/batch/pom.xml

```diff
@@ -42,7 +42,7 @@
     <maven-surefire-plugin.version>3.0.0</maven-surefire-plugin.version>
     <nemo.version>0.1</nemo.version>
     <flink.artifact.name>beam-runners-flink-1.14</flink.artifact.name>
-    <mysql.driver.version>8.0.16</mysql.driver.version>
+    <mysql.driver.version>8.0.33</mysql.driver.version>
```

Comment:
Can you please elaborate how this version upgrade solves the GCS issues?

Reply:
The dependency jar `com.google.protobuf:protobuf-java` in the `mysql:mysql-connector-java` was incompatible with the latest `com.google.api.grpc:proto-google-common-protos` jar.  This was causing the application to fail during launch if the parquet file location is GCS. This has been excluded in the latest [commit](https://github.com/google/fhir-data-pipes/pull/678/commits/5d04f541237496a5651f2474e6b5db422c0d691c)